charts/azuremonitor-containers/templates/omsagent-daemonset-windows.yaml

```diff
@@ -53,6 +53,12 @@ spec:
        resources:
 {{ toYaml .Values.omsagent.resources.daemonsetwindows | indent 9 }}
        env:
+       - name: FBIT_SERVICE_FLUSH_INTERVAL
```

Comment:
this was missed in last release ? or is it that we never released for long time since this was added on AKS side

Reply:
> this was missed in last release ? or is it that we never released for long time since this was added on AKS side

> This is for windows agent and which targeted for this release with FBIT upgrade.

Reply:
ok got it.